client/src/cbltest/api/remoteshell.py

```diff
@@ -0,0 +1,183 @@
+import subprocess
```

Comment:
I'm very skeptical about having tests that actually connect to and execute commands on a remote machine because now there are assumptions here about the environment on the other side.  I guess there might be no other way to do this for edge server but at least I'd like to keep this as minimal as possible.  Having a class called "remote shell" is vague and hard to understand.